### ISSUE TITLE
Can't get basic features to work

### ISSUE THREAD

**Issue by jordi-chacon** [rengwuxian/MaterialEditText#30]
Hi!

First of all I would like to thank you for contributing to the Android community.

I just started using your library (version 1.6.0 via gradle) for my application and I can't seem to get basic features to work.

My xml:

``` xml
<LinearLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/tools"
    android:layout_height="match_parent"
    android:layout_width="match_parent"
    android:orientation="vertical"
    android:padding="16dp">

    <com.rengwuxian.materialedittext.MaterialEditText
        android:id="@+id/name_material_edit_text"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:hint="@string/name"
        android:textCursorDrawable="@drawable/cursor_accent"
        app:baseColor="@android:color/black"
        app:floatingLabel="highlight"
        app:primaryColor="@color/accent" />

</LinearLayout>
```

Then I have a super simple activity which sets its content view to the xml above and does nothing more.

When I run the activity this is what I get:
- an edit text, focused, with a hint "Name" with light gray. This is expected.
- even though the edit text is focused, the horizontal line is black instead of using my accent color. This is not expected.
- as I start typing, there is no floating label. This is not expected.

I have also tried setting an error by calling mEditText.setError("error text"); but that didn't display any error.

So basically, almost nothing is working for me :/

I am running 4.4.2 in a Nexus 5. I tried your sample apk and that seems to work just fine. I get all the nice features, like floating label, errors, colouring of horizontal line and so on.

Do you know what I might be doing wrong?

Thanks again!
